app/javascript/gobierto_data/webapp/components/sets/SQLEditorCode.vue

```diff
@@ -75,7 +79,7 @@ export default {
         mode: 'text/x-sql',
         hintOptions: {
           completeSingle: false,
-          tables: {}
+          hint: this.hint
```

Comment:
What is `this.hint`? Looking the code is null. Do not initialize `data` variables with another `data` variable value. Perhaps a `prop`, but we should use simple types

Reply:
`This.hint` activate autocomplete for SQL and Columns https://github.com/PopulateTools/gobierto/blob/902_sql_editor_autocomplete/app/javascript/gobierto_data/webapp/components/sets/SQLEditorCode.vue#L215

Reply:
I see. That's due to the same I tell you [here](https://github.com/PopulateTools/gobierto/pull/2843#discussion_r378160857)

Reply:
Even more, do not nest objects in `data()`. Every single time you do so, Vue creates observer methods for each property. In a nested object, you create observers for `hintOptions`, and then por `completeSingle`, `tables`, and so on.

That's a bad practice, who penalizes performance